tools/clang-format-diff.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python
+
+# BLURB gpl
+#
+#                           Coda File System
+#                              Release 6
+#
+#          Copyright (c) 1987-2018 Carnegie Mellon University
```

Comment:
You cannot just slap a CMU GPL-v2 license on top of someone else's code. This looks like it came from,
https://github.com/llvm-mirror/clang/blob/master/tools/clang-format/clang-format-diff.py

So the license looks like a BSD one, https://github.com/llvm-mirror/clang/blob/master/LICENSE.TXT.

First of all, wouldn't this be installed somewhere already along with clang-format? (second of all, although nice I don't think we really need this)

Reply:
I was aware of the license and I thought that I could re-license it because it's allowed to sublicense it.
I checked for the presence of `clang-format-diff.py` in my system before and didn't find it because it's actually installed as `clang-format-diff`.
I still think that the functionality of the script is needed. Please read the comments in the pre-commit hook. I'm sorry for the misunderstanding. ;)